pkg/configuration/configuration.go

```diff
@@ -93,6 +145,16 @@ func (c *Config) GetDurationBeforeNotificationDeletion() time.Duration {
 	return c.host.GetDuration(varDurationBeforeNotificationDeletion)
 }
 
+// GetMailgunDomain returns the host operator mailgun domain
+func (c *Config) GetMailgunDomain() string {
+	return c.host.GetString(varMailgunDomain)
+}
+
+// GetMailAPIKey returns the host operator mailgun api key
+func (c *Config) GetMailgunAPIKey() string {
+	return c.host.GetString(varMailgunAPIKey)
+}
+
```

Comment:
If we are exposing the mailgun configuration here, we also need to include the sender email address.  Something like `GetMailgunSenderEmail()`.

Reply:
hmm, i don't think this should be part of this story? this story was specific to setting up the secret for mailgun (https://issues.redhat.com/browse/CRT-697). i think this should be part of the story that handles actually sending the emails..

Reply:
added here: d1250e5db929e62d1956927d47ae30ef533a9235